modules/KoreBuild.Tasks/InstallDotNet.cs

```diff
@@ -0,0 +1,259 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Build.Framework;
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace KoreBuild.Tasks
+{
+    /// <summary>
+    /// Shim for executing dotnet-install.{sh,ps1}, allowing repos to specify the required .NET Core runtimes or to programmatically add new ones.
+    /// </summary>
+    public class InstallDotNet : Microsoft.Build.Utilities.Task, ICancelableTask
+    {
+        public readonly CancellationTokenSource _cts = new CancellationTokenSource();
```

Comment:
This kind of duplicates some of what I'm doing in #363 but having it as a task is nice, and we could change this to call the KoreBuild command later on.

Reply:
We can take a look at that later on.

Reply:
Yeah, more of a note than anything.